rust/core-lib/src/plugins/catalog.rs

```diff
@@ -14,6 +14,8 @@
 
 //! Keeping track of available plugins.
 
+#![allow(clippy::redundant_clone)]
```

Comment:
I think clippy is actually pointing out an actual error here; we have a `to_owned()` call on L119 of this file that we can remove.

Reply:
Looks feasible, feedback later.